mslib/utils/auth.py

```diff
@@ -100,14 +98,21 @@ def get_auth_from_url_and_name(server_url, http_auth, overwrite_login_cache=True
         if server_url == url:
             try:
                 password = get_password_from_keyring(service_name=url, username=auth_name)
+                if password is None:
```

Comment:
We don't allow to save empty strings as password, that is not a good idea, to remove None here.